src/game/hud.c

```diff
@@ -599,9 +599,6 @@ void render_hud(void) {
             render_hud_timer();
         }
 
-        if (gSurfacePoolError & NOT_ENOUGH_ROOM_FOR_SURFACES) print_text(10, 40, "SURFACE POOL FULL");
```

Comment:
We definitely still need some way of telling the user when the dynamic pool is full.